.github/workflows/release-shotcaller-messaging-pipeline.yml

```
@@ -56,4 +56,7 @@ jobs:
       dotnet-version: ${{ inputs.dotnet_version }}
       package-version: ${{ needs.release.outputs.VersionNumber }}
       project-name: ShotCaller.Azure.ServiceBus.Messaging
-      release-notes-url: https://github.com/Cheranga/shotcaller/releases/tag/${{ needs.release.outputs.VersionNumber }}
\ No newline at end of file
+      release-notes-url: https://github.com/Cheranga/shotcaller/releases/tag/${{ needs.release.outputs.VersionNumber }}
+    secrets:
+      NUGET_KEY: ${{ secrets.NUGET_KEY }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)